src/node/alias.ts

```diff
@@ -17,47 +15,24 @@ export const DEFAULT_THEME_PATH = join(DIST_CLIENT_PATH, 'theme-default')
 export const SITE_DATA_ID = '@siteData'
 export const SITE_DATA_REQUEST_PATH = '/' + SITE_DATA_ID
 
-const vueRuntimePath = 'vue/dist/vue.runtime.esm-bundler.js'
-
 export function resolveAliases(root: string, themeDir: string): AliasOptions {
   const paths: Record<string, string> = {
-    '/@theme': themeDir,
     '@theme': themeDir,
     [SITE_DATA_ID]: SITE_DATA_REQUEST_PATH
   }
 
-  // prioritize vue installed in project root and fallback to
```

Comment:
This logic is important and should not be removed.

We can use [createRequire](https://nodejs.org/api/module.html#modulecreaterequirefilename) to preserve the same logic in esm.